package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "last-train-to-nodeville",
+  "version": "1.0.0",
+  "description": "## Instructions",
+  "main": "index.js",
+  "dependencies": {
+    "chai": "^4.2.0",
+    "mocha": "^6.2.0"
+  },
+  "devDependencies": {},
```

Comment:
Expected to see mocha and chia specified here as dev dependencies.

Reply:
Added Mocha and Chai to Dev Dependencies
![image](https://user-images.githubusercontent.com/32113320/65367056-ec576180-dbf9-11e9-8e51-1ef7f7a3f83f.png)

Reply:
Saved Mocha and Chai to Dev Dependencies and they were removed from "dependencies"